### DISCUSSION TITLE
Can multiple useMDXComponent be used safely?

### DISCUSSION THREAD

**Discussion by khinshankhan** [contentlayerdev/contentlayer#404] [Q&A]
I'm making a list of mdx content. Each item in the list ends up needing to use `useMDXComponent` for the card. Is `useMDXComponent` expensive? Is it alright to use it multiple times on a page?

**Comment by git-no** [comment]
@khinshankhan Hi, I am not quite sure, if this is an issue. It sounds more as a question, than an error.
Maybe instead of asking help in an issue ticket you want to ask your question in the Contentlayer [Discord Channel](https://discord.gg/fk83HNECYJ) - probably you get an response / answer much quicker.

**Comment by khinshankhan** [comment]
Oh I see. I joined and asked, thanks :+1:
